metrics/nodehealthcheck.go

```diff
@@ -42,10 +44,19 @@ var (
 	)
 )
 
+var nodehealtCheckRemediationDuration = prometheus.NewHistogramVec(
```

Comment:
no clue why the other vars are public, but I would like to have it consistent please (either way)